examples/Dockerfile

```diff
@@ -22,6 +22,9 @@ awk \
 iptables \
 cronie \
 openssh \
+audit \
```

Comment:
Please add the same to README.md, like below:
https://github.com/vmware/ansible-security-hardening/pull/10/files